requirements.txt

```diff
@@ -1,7 +1,7 @@
 adal==0.4.7
 argcomplete==1.8.0
-colorama==0.3.7
-humanfriendly==2.4
+colorama==0.3.9
+humanfriendly==4.7
```

Comment:
Why do you need to bump these versions?

Reply:
I chatted with @troydai and we decided to modify the version after verifying that no strange behaviors happen after bumping up.

Reply:
I asked @chewong to look into this. This library has been changed a bit, it worths to pull in the latest.

Reply:
Update `setup.py` to reflect the version range selection of `humanfriendly`.